tendrl/node_agent/monitoring/collectd/collectors/gluster/low_weight/tendrl_glusterfs_health_counters.py

```diff
@@ -1,221 +1,144 @@
+    def get_volume_snap_counts(self):
+        ret_val = {}
+        volumes = self.CLUSTER_TOPOLOGY.get('volumes', [])
+        for volume in volumes:
+            snap_cnt = volume.get('snap_count', 0)
+            ret_val[volume['name']] = snap_cnt
+        return ret_val
 
+    def get_vol_rebalance_bytes(self):
+        ret_val = {}
+        volumes = self.CLUSTER_TOPOLOGY.get('volumes', [])
+        for volume in volumes:
+            rebalance_data = volume.get('rebalance_data', 0)
+            ret_val[volume['name']] = rebalance_data
+        return ret_val
 
-def get_volume_status_wise_counts():
-    global CLUSTER_TOPOLOGY
-    volumes = CLUSTER_TOPOLOGY.get('volumes', [])
-    volume_total_cnt = len(volumes)
-    volume_up_count = 0
-    for volume in volumes:
-        if volume.get('status') == 'Started':
-            volume_up_count = volume_up_count + 1
-    return (
-        volume_total_cnt,
-        volume_total_cnt - volume_up_count,
-        volume_up_count
-    )
+    def get_vol_rebalance_files(self):
+        ret_val = {}
+        volumes = self.CLUSTER_TOPOLOGY.get('volumes', [])
+        for volume in volumes:
+            rebalance_files = volume.get('rebalance_files', 0)
+            ret_val[volume['name']] = rebalance_files
+        return ret_val
 
+    def get_volume_rebalance_failures(self):
+        ret_val = {}
+        volumes = self.CLUSTER_TOPOLOGY.get('volumes', [])
+        for volume in volumes:
+            rebalance_failures = volume.get('rebalance_failures', 0)
+            ret_val[volume['name']] = rebalance_failures
+        return ret_val
 
-def get_volume_brick_statuses(volume):
-    ret_val = {}
-    for sub_volume_index, sub_volume_bricks in volume.get(
-        'bricks',
-        []
-    ).iteritems():
-        for brick in sub_volume_bricks:
-            brick_hostname = brick.get('hostname')
-            if (
-                brick_hostname == socket.gethostbyname(
-                    CONFIG['peer_name']
-                ) or
-                brick_hostname == CONFIG['peer_name']
-            ):
-                if 'status' in brick:
-                    brick_statuses = ret_val.get(brick['hostname'], [])
-                    status = tendrl_glusterfs_utils.get_brick_state_mapping(
-                        brick['status']
-                    )
-                    brick_statuses.append(
-                        {
-                            'status': status,
-                            'path': brick['path']
-                        }
-                    )
-                    ret_val[brick['hostname']] = brick_statuses
-    return ret_val
+    def get_volume_rebalance_skipped(self):
+        ret_val = {}
+        volumes = self.CLUSTER_TOPOLOGY.get('volumes', [])
+        for volume in volumes:
+            rebalance_skipped = volume.get('rebalance_skipped', 0)
+            ret_val[volume['name']] = rebalance_skipped
+        return ret_val
 
+    def get_volume_status_wise_counts(self):
```

Comment:
Make these function private if not called from outside.

Reply:
ok

Reply:
Done